Podfile

```diff
@@ -49,17 +49,29 @@ target 'WordPressAuthenticatorTests' do
   pod 'OCMock', '~> 3.4'
   pod 'Expecta', '1.0.6'
   pod 'Specta', '1.0.7'
+end
+
+abstract_target 'CI' do
+  pod 'SwiftLint', '~> 0.49'
+end
```

Comment:
Hmm, what is this black magic? Would adding this abstract target alone make cocoapods to install SwiftLint during CI checks?

Reply:
Just tried `pod install` locally. So, what this abstract target does is installing a SwiftLint pod but no using it in any targets, isn't it?

> [!] The abstract target CI is not inherited by a concrete target, so the following dependencies won't make it into any targets in your project:
    - SwiftLint (~> 0.49)

Reply:
> So, what this abstract target does is installing a SwiftLint pod but no using it in any targets, isn't it?

Oh! I see now that calling the target "CI" might be a bit confusing, in particular when I wrote in the PR description that this doesn't interact with CI 🤦‍♂️ 

> what this abstract target does is installing a SwiftLint pod but no using it in any targets, isn't it?

Yes. It's a bit of a misuses of [`abstract_target`](https://guides.cocoapods.org/syntax/podfile.html#abstract_target) which is meant for "convenient target dependency inheritance", but it does the job of downloading the binary at the specified version without the need of explicitly adding it to any of the targets because we don't need it at run time, just as a tool during build time.

Reply:
I'll rename "CI" to "Tools", which might make this a bit clearer.